bin/internal/engine.version

```diff
@@ -1 +1 @@
-248290d6d50a1bf1a6a246544347c03acda867a5
+0f359063c487ee70787e58b4b011cbb3c2f53fd6
```

Comment:
```suggestion
0f359063c487ee70787e58b4b011cbb3c2f53fd6

```

Flutter style guide requires NL before EOF

Reply:
@itsjustkevin mentioned that an extra line in the engine.version file might cause problems but looking at what we have in master I think we can safely add the extra line